frontend/src/components/Home.js

```diff
@@ -2,8 +2,148 @@ import React from "react"
 
 function Home() {
     return (
-        <div>
-            [Implement landing page here]
+        <div id="landing-content">
+            <div className="container">
+                <div className="columns">
+                    <div className="column">
+                        <h1 className="title is-size-1 has-text-weight-bold" style={{paddingBottom: 100}}>
+                           Entrena con quien <br></br>quieras y donde sea
+                        </h1>
+                        <ul>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Con tu amigo, profe o compañero</span>
+                            </li>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Enterate de nuevas clases a <br></br> precios increíbles, y hasta <br></br> gratis.</span>
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="column">
+                        <figure className="image is-3by2">
+                            <img src="https://bulma.io/images/placeholders/256x256.png"  alt="demo.jpg" />
+                        </figure>
+                        <p className="has-text-right">&quot;Al fin voy al gimnasio, gracias a mi amiga&quot;&nbsp;</p>
+                        <p className="has-text-right">Cristina&nbsp;&nbsp;</p>
+                    </div>
+                </div>
+                <div className="columns">
+                    <div className="column has-text-centered">
+                        <a href="/signup"  className="button is-large register-btn">registrate</a>
+                    </div>
+                </div>
+                <div className="columns">
+                    <div className="column">
+                        <h1 className="title">Las clases de hoy</h1>
+                        <p className="subtitle">Todos los días se suman nuevas clases</p>
+                        <div className="columns">
+                            <div className="column">
+                                <figure className="image is-2by1" style={{marginBottom: 10}}>
```

Comment:
extract content inside the column to a separate Component. something named `Activity` or similar. the component should read the title `Rock cicle` and the description `Lorem ...` from `props` object. Make it a functional component, please avoid inheriting from Component base class. This component will probably be reused in many places.

It might be a good idea to the same for Gyms, so also extract that part of HTML too.

instead of hardcoding the information directly in html, let's make that information come from somewhere else. keep in mind that eventually it will come from the API. let's use `state` and `effect` hooks to do this:
- https://reactjs.org/docs/hooks-state.html
- https://reactjs.org/docs/hooks-effect.html

declare some state for the home page, and set that state inside the effect hook. the state should be something like:
```json
{
  "activities": [
    { "name": "Rock cycle", "description": "Lorem ...", "img": "some-image-path" },
    { "name": "Rock cycle", "description": "Lorem ...", "img": "some-image-path" }
  ],
  "gyms": [
    { "name": "Octava compania", "img": "some-image-path" },
    { "name": "Octava compania", "img": "some-image-path" },
    { "name": "Octava compania", "img": "some-image-path" }
  ]
}
```

once that's set, make use of the state variable to render these parts of the HTML using the extracted components.